packages/modal/README.md

```diff
@@ -78,13 +78,14 @@ function ExampleComponent() {
+| `open`                 | `boolean`                                                                                | Determines open state of `Modal` component                                                                                                                                                         | `false`      |
+| `setOpen`              | `function`                                                                               | Callback to set open state of Modal component. `setOpen` accepts a boolean value, which will determine the open state of `Modal` component.                                                        | `() => {}`   |
+| `size`                 | `'small'`, `'default'`, `'large'`                                                        | Determines `Modal` size.                                                                                                                                                                           | `'default'`  |
+| `shouldClose`          | `function`                                                                               | Callback to determine whether or not Modal should close when user tries to close it.                                                                                                               | `() => true` |
+| `children`             | `node`                                                                                   | Children that will be rendered inside `<Modal />` component.                                                                                                                                       |              |
+| `className`            | `string`                                                                                 | Style to be applied to the container's root node.                                                                                                                                                  |              |
+| `contentClassName`     | `string`                                                                                 | Style to be applied to the content div.                                                                                                                                                            |              |
+| `closeOnBackdropClick` | `boolean`                                                                                | Determines whether or not a Modal should close when a user clicks outside the modal.                                                                                                               | `true`       |
+| `initialFocus`         | `string` (Selector that can be passed to document.querySelector() to find the DOM Node.) | By default, when a focus trap is activated the first element in the focus trap's tab order will receive focus. With this option you can specify a different element to receive that initial focus. |              |
```

Comment:
I'd make this bit of documentation a bit less specific to the implementation. I would not refer to the focus trap, and instead say something like: 
> A selector string for the element to move focus to when the modal is opened. The first focusable element in the modal will receive focus by default.

Then you don't need additional detail in the `type` column as well.

Reply:
Sounds good, thanks